setup.py

```diff
@@ -46,9 +46,10 @@
 # for compatibility reasons.
 
 DEPENDENCIES = [
-    "paho-mqtt==1.5.0",
```

Comment:
We should delete the paho-mqtt license section from the `thirdpartynotice` file since we would take a dependency on it through the device SDK which includes the third party notice.